### DISCUSSION TITLE
describe image into prompt - model

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#3202] [Q&A] [answered]
a question

is it possible to use other models for this, eg llava1.6 with the mmproj file?
or is that totaly different?
is it easy to implement or you have the intention to doo?

**Comment by mashb1t** [comment, marked as answer]
BLIP is currently the only supported model for image description, and i've atarted to compare it to others but no final results yet: https://github.com/mashb1t/describeiments

What Fooocus is not going to be is a tool interacting with LLMs.

**Comment by kalle07** [comment]
i see ... so

**Comment by kalle07** [comment]
i see, so
blip2-opt2.7b-coco
git-large-coco
and so on...
you compare them in function or in qualitiy?
so llava is llm but it comes 80% close to gpt4  ;)

**Reply by mashb1t** [reply]
Yes, that's at least the plan, but after https://github.com/mashb1t/Fooocus/releases/tag/v2.5.0 and https://github.com/mashb1t/Fooocus/releases/tag/v2.5.1 i'm taking a small break + am currently sick, so development of this feature has to wait.

**Reply by kalle07** [reply]
Get well soon!
